code/modules/mob/living/carbon/human/human.dm

```diff
@@ -327,6 +327,11 @@
 		else
 			dat += "<tr><td><B>Eyes:</B></td><td><A href='?src=[UID()];item=[slot_glasses]'>[(glasses && !(glasses.flags&ABSTRACT))	? html_encode(glasses) : "<font color=grey>Empty</font>"]</A></td></tr>"
 
+		if(slot_neck in obscured)
+			dat += "<tr><td><font color=grey><B>Neck:</B></font></td><td><font color=grey>Obscured</font></td></tr>"
+		else
+			dat += "<tr><td><B>Neck:</B></td><td><A href='?src=[UID()];item=[slot_neck]'>[(neck && !(neck.flags&ABSTRACT))	? html_encode(neck) : "<font color=grey>Empty</font>"]</A></td></tr>"
+
```

Comment:
I think this would work much better further up in the list, since at the moment it's between the 'Eyes' and 'Left Ear' slots which is a bit unintuitive.
![image](https://user-images.githubusercontent.com/57483089/126906731-7e467b6c-21b1-4bbb-9dad-e5bc8a404657.png)
Maybe between 'Back' and 'Head', or under 'Mask'.